website_sale_search_fuzzy/hooks.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, SUPERUSER_ID
+from odoo.exceptions import UserError
+
+
+def _add_trgm_index_product_tmpl_name(cr, registry):
+    with cr.savepoint():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        trgm_mod = env['trgm.index']
+
+        if trgm_mod._trgm_extension_exists() != 'installed':
+            raise UserError(_(
```

Comment:
@lasley I'm doing `env['trgm.index'].create({'field_id': prod_templ_name, 'index_type': 'gin'})` below here in this post init hook, adding the index to the product template name field.

I have a check here that ensures trgm is installed on postgres (used from base_search_fuzzy) before adding the index, however I don't think travis or runbot are configured to have it installed.

Is doing a post_init_hook the wrong way to go about it? Or not sure if some config files have to be edited to get trgm installed.

Perhaps doing the post_init_hook is correct and having a trgm check is unnecessary.